<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSelectTab</class>
 <widget class="QWidget" name="FileSelectTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>636</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FileSelectTab</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="directoryLayout">
     <item>
      <widget class="QPushButton" name="openButton">
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="text">
        <string>Verzeichnis...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditDirectory">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="treeView"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>treeView</tabstop>
  <tabstop>openButton</tabstop>
  <tabstop>lineEditDirectory</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
